tests/stress/README.md

```diff
@@ -0,0 +1,94 @@
+# Stress Testing
+
+## Intro
+To stress test csvcubed, we use a tool called JMeter to run two separate test plans for the build command and inspect command respectively.  
+The test plans are run consecutively via a bash script and follow broadly the same structure:
+
+- Start running a performance monitor in the background
+- Generate a "maximally complex" CSV file with the number of rows the user has given as a parameter to the bash script.
+  1. This is a preprocess which is a groovy script which in turn runs a python script because JMeter will not execute `.py` files.
+      - The inspect command test then runs 1 build command on this CSV file so that a `.csv-metadata.json` file can be used.
```

Comment:
Meant to say "The inspect command test then runs the build command on this CSV file so that the json-ld output generated by the build command can be used as the input to the inspect command"?

Reply:
Changed wording. This is far more succinct - thank you